third_party/libunwindstack/utils/RegsFake.h

```diff
@@ -102,7 +104,6 @@ class RegsImplFake : public RegsImpl<TypeParam> {
     *value = fake_pseudo_reg_value_;
     return true;
   }
-
```

Comment:
Nit: Not sure if that change was intended.

Reply:
No, not intended, thanks for spotting. Reverted this part.